### PR TITLE
Result page styling bug#439/zubkov

### PR DESCRIPTION
**Pull request by olegz7**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -18,6 +20,7 @@ export class AppComponent implements OnInit{
 
   isWindowMobile(event: any): void {
     this.MobileView = event.innerWidth <= 750;
+    this.store.dispatch(new ToggleMobileScreen(this.MobileView))
```
Comment (litvinets) [inline src/app/app.component.ts:23]: I am not sure this functionality should be implemented in the app component. I believe it should be moved to header or result components.

File: src/app/shared/components/scroll-to-top/scroll-to-top.component.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+@Component({
+  selector: 'app-scroll-to-top',
+  templateUrl: './scroll-to-top.component.html',
+  styleUrls: ['./scroll-to-top.component.scss']
+})
+export class ScrollToTopComponent implements OnInit {
+
+  showScrollButton: boolean;
+  topPosToShowButton = 100;
```
Comment (litvinets) [inline src/app/shared/components/scroll-to-top/scroll-to-top.component.ts:14]: I think it should be moved as constant to constants.ts

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 1 above)
Comment (dmitryy90) [reply src/app/app.component.ts:23]: it was decided to move it from header to app component to have the possibility to pass isMobile flag to multiple child components without creating additional subscriptions

File: src/app/shared/components/scroll-to-top/scroll-to-top.component.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  position: fixed;
+  bottom: 25px;
+  left: 45%;
+  border: 1px solid #3849F9;
+  border-radius: 50%;
+  outline: none;
+  padding: 0 !important;
+  background-color: #3849F9;
+  color: white;
+  &.mat-button {
```
Comment (dmitryy90) [inline src/app/shared/components/scroll-to-top/scroll-to-top.component.scss:11]: indentations

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**